src/pages/PodcastAlbumCollectionAdminPage.tsx

```diff
@@ -0,0 +1,80 @@
+import Icon, { FileAddOutlined } from '@ant-design/icons'
+import { useMutation } from '@apollo/react-hooks'
+import { Button } from 'antd'
+import gql from 'graphql-tag'
+import { useEffect } from 'react'
+import { useIntl } from 'react-intl'
+import { useHistory } from 'react-router-dom'
+import { AdminPageTitle } from '../components/admin'
+import ProductCreationModal from '../components/common/ProductCreationModal'
+import AdminLayout from '../components/layout/AdminLayout'
+import PodcastAlbumCollectionTabs from '../components/podcastAlbum/PodcastAlbumCollectionTabs'
+import { useApp } from '../contexts/AppContext'
+import { useAuth } from '../contexts/AuthContext'
+import hasura from '../hasura'
+import { handleError } from '../helpers'
+import { commonMessages, podcastAlbumMessages } from '../helpers/translation'
+import { ReactComponent as MicrophoneIcon } from '../images/icon/microphone.svg'
+
+const PodcastAlbumCollectionAdminPage: React.VFC = () => {
+  const { formatMessage } = useIntl()
+  const { id: appId } = useApp()
+  const { currentMemberId } = useAuth()
+  const history = useHistory()
+  const [createPodcastAlbum] = useMutation<hasura.INSERT_PODCAST_ALBUM, hasura.INSERT_PODCAST_ALBUMVariables>(
+    INSERT_PODCAST_ALBUM,
+  )
+
+  // clean up
+  useEffect(() => {
+    return () => {}
+  }, [])
```

Comment:
why need to clean up here ?

Reply:
try to fix
```
Warning: Can't perform a React state update on an unmounted component.
This is a no-op, but it indicates a memory leak in your application.
To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
```

Reply:
update the state of the unmounted component
```jsx
const [modal, setModal] = useState(true)

{
modal && <Modal renderTrigger={({ setVisible }) => {
     return <button onClick={() => {
        setModal(false)
        setTimeout(() => setVisible(true), 100)
    }}>Open</button>
}}>
}
```